### ISSUE TITLE
About M3FD results

### ISSUE THREAD

**Issue by 225ceV** [XueZ-phd/TFDet#6]
Hello author, I have doubts about the results of M3FD in the paper. I think it is because TarDAL did not provide the meta information of the data set and the code is incomplete. However, I was unable to reach the TarDAL author via github, considering that you may have replicated the code, and the TarDAL method metrics in the paper are different from those in the original paper.

The following are the problems I have encountered, and I would appreciate your advice:

1. I tried to retrain TarDAL, but the fusion model with its original hyperparameters did not seem to converge, and the published code seemed to be inconsistent with the paper in several places.
![image](https://github.com/user-attachments/assets/add699d3-052e-4c6a-b568-c7c0cb61a737)
From the figure, it seems that the restruction loss(fus/src) of the fusion module is the crux of the problem. All other losses were on the order of 10^0
2. I tried to use its pre-training weight for evaluation, but found that it did not include the weight of the object detection model. I would like to know how you got the comparison data in your paper, is it freezing fusion model training object detection model?

**Comment by XueZ-phd** [comment]
We recommend that you communicate with the author of TarDAL.
